page-impact.hbs

```diff
@@ -0,0 +1,550 @@
+{{!< default}}
+
+<div class="gh-page">
+    <div class="gh-container">
+
+        <!-- IMPACT-OUR-IMPACT -->
+        <div class="row">
+            <div class="col-xs-12 col-md-12 col-lg-12 mb-5">
+                <h1>Our Impact</h1>
```

Comment:
Could you shift this into a `header` class instead, similar to this snippet of code:
```
<header class="gh-header gh-canvas">
    <h1 class="gh-title">Contact Us</h1>
</header>
```

This follows the way Ghost renders pages created through the editor in the admin panel.